### ISSUE TITLE
Java Language Frontend: J_CONSTR_BEGIN, J_CONSTR_END and J_VOID never used

### ISSUE THREAD

**Issue by euberdeveloper** [jplag/JPlag#1865]
It can be seen from the code that these token types for Java are never used:

![image](https://github.com/jplag/JPlag/assets/33126163/54beb248-f8d5-4329-b7a9-14fe8c805d8e)

**Comment by euberdeveloper** [comment]
The same can be said for other tokens

![image](https://github.com/jplag/JPlag/assets/33126163/a9f5ce42-5375-475e-a0cf-792639122175)

**Comment by tsaglam** [comment]
Yes, the unused tokens and the comments after each line are very much legacy. Some of them should not be used during tokenization as they decrease detection quality by enlarging the attack surface. For others, it is unclear if they have an impact at all.

**Comment by Kr0nox** [comment]
@TwoOfTwelve I think you mentioned something about unused tokens in one of our meetings? Is this the same? Did they get deleted now?

**Comment by TwoOfTwelve** [comment]
It is the same. They are not deleted yet. I'll investigate why they are here and open a PR
